plugins/modules/fusion_se.py

```diff
@@ -238,20 +273,37 @@ def create_se(module, fusion):
     se_api_instance = purefusion.StorageEndpointsApi(fusion)
 
     if not module.check_mode:
-        discovery_interfaces = [
-            purefusion.StorageEndpointIscsiDiscoveryInterfacePost(**endpoint)
-            for endpoint in module.params["iscsi"]
-        ]
-        storage_endpoint = purefusion.StorageEndpointPost(
-            name=module.params["name"],
-            display_name=module.params["display_name"] or module.params["name"],
-            endpoint_type=module.params["endpoint_type"],
-            iscsi=purefusion.StorageEndpointIscsiPost(
-                discovery_interfaces=discovery_interfaces
-            ),
+        iscsi = (
+            purefusion.StorageEndpointIscsiPost(
```

Comment:
This is probably going to be confusing for people who are not well versed in python, because it is not immediately apparent this is actually ternary operator. It took me a while to parse this as well. Personally I would just stick to
```
iscsi = None
if module.params["endpoint_type"] == "iscsi":
  iscsi = ...
```
because that is obvious.

Reply:
addressed